src/views/sign-in/external/KB_SSO/template/KB_SSO.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+    <div class="kbSSO-wrapper">
+        <form id="myForm" @submit.prevent="openKBSSO">
+            <p-button
+                style-type="primary1 outline"
+                size="lg"
+                class="kbSSO-btn"
+            >
+                KB SSO
+            </p-button>
+        </form>
+    </div>
+</template>
+
+<script lang="ts">
+import axios from 'axios';
+import {
+    ComponentRenderProxy, defineComponent, getCurrentInstance,
+} from '@vue/composition-api';
+import { PButton } from '@spaceone/design-system';
+import { loadAuth } from '@/views/sign-in/authenticator/loader';
+
+export default defineComponent({
+    name: 'KBSignIn',
+    components: {
+        PButton,
+    },
+    setup(props, { context, emit }) {
+        const vm = getCurrentInstance() as ComponentRenderProxy;
+        const openKBSSO = async () => {
+            try {
+                window.location.href = '/checkauth.jsp?secureToken=zzzzzz&secureSessionId=xxxx&resultCode=200OK';
```

Comment:
이 부분은 위에서 만들어두신  `SIGN_IN_ROUTE.KB._NAME `으로 `router.push` 를 통해 페이지 라우팅을 하면 좋을 것 같습니다!
이유는, 필요한 query 에 대한 스펙 관리 이슈, 페이지 라우팅 라이프사이클 이슈 등을 수월하게 관리하기 위함입니다.

```suggestion
  await vm.$router.push({
                    name: SIGN_IN_ROUTE.KB._NAME,
                    query: {
                        secureToken: 'zzzzzz',
                        secureSessionId: 'xxxx',
                        resultCode: '200OK',

                    },
                });
```

Reply:
해당부분은 최초에 'http://kbpkiapc.kbstar.com:9080/api/v1/sso/checkserver' 를 저희가 호출하면 해당 페이지에서'/checkauth.jsp?secureToken=zzzzzz&secureSessionId=xxxx&resultCode=200OK' 로 redirect 해줍니다.. 해당 주소는 내부망에서만 호출이 되기때문에 현재는 작업을 할 수 없어 임의로 href로 작업 해놓았습니다.

Reply:
아 그러면 
window.location.href = '/checkauth.jsp?secureToken=zzzzzz&secureSessionId=xxxx&resultCode=200OK';
이 부분은, 최초에  'http://kbpkiapc.kbstar.com:9080/api/v1/sso/checkserver' 를 부르는 것을 임의로 테스트하려고 넣어두신 거라는 말씀이시죠??

Reply:
앗 그러면 리다이렉트되는 url 형태가
`/checkauth.jsp?secureToken=zzzzzz&secureSessionId=xxxx&resultCode=200OK`
라면,  `kbSSORoute` 를 아래처럼 바꾸면 어떨까요?

```

export const kbSSORoute = {
    path: '/checkauth.jsp',
    name: SIGN_IN_ROUTE.KB._NAME,
    meta: {
        excludeAuth: true,
        isSignInPage: true,
    },
    props: ({ query }) => ({
        secureToken: query.secureToken,
        secureSessionId: query.secureSessionId,
        resultCode: query.resultCode,
        nextPath: query.nextPath || '/',
    }),
    component: KB_SSO,
} as RouteConfig;

```

Reply:
작업 완료되었습니다~!

Reply:
와우 고생많으셨습니당 ! ㅎㅎ